modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/LayoutImportController.java

```diff
@@ -653,7 +653,8 @@ else if (ExportImportThreadLocal.isStagingInProcess() &&
 		List<Locale> sourceAvailableLocales = Arrays.asList(
 			LocaleUtil.fromLanguageIds(
 				StringUtil.split(
-					headerElement.attributeValue("available-locales"))));
+					headerElement.attributeValue("available-locales")),
```

Comment:
@marianoalvarosaiz The logic here can be simplified. First, you translated langaueId into local, then you check LanguageUtil.isAvailableLocale(), if you go into this method, you will see it translate local into languageId to do the checking.
We should avoid this kind of translation.

Tina